Easy/p200_to_210.py

```diff
@@ -58,6 +58,30 @@ def next_number(n_):
 print(isHappy_mkf(99))
 
 
+def isHappy_fati1(n) -> bool: ## Run time error for some cases!
+    try:
+        if n == 1:
+            return True
+        else:
+            n = sum(list(map(lambda x: pow(int(x), 2), list(str(n)))))
```

Comment:
How do you handle a loop? you can have a loop of values! for example:
22 -> 8 -> 64 -> 52 -> **29** -> 85 -> 89 -> 145 -> 42 -> 20 -> 4 -> 16 -> 37 -> 54 -> 41 -> 17 -> 50 -> 25 -> **29** (agin 29 and the loops continues)

Reply:
You are right Mohammad. I will remove this function.